draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,84 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
+          <t>Note: The following algorithm does not yet have WG
+          consensus but is included here as something concrete for the
+          working group to discuss."</t>
+          <t>When an RTP packet is received by a transport, that
+          packet needs to be routed to the correct RtpReceiver. For
+          each transport, the following steps MUST be followed to
+          prepare to route packets:</t>
+
+          <t>
+            <list>
+              <t>Construct a table mapping MID to RtpReceiver. For
+              each RtpReceiver configured to receive from this
+              transport, let the key be the RtpReceiver's MID, and the
+              value be the RtpReceiver.  The MID of a given
+              RtpReceiver is found in the a=mid line of the m= section
+              corresponding to that RtpReceiver in remote local and
+              remote descriptions.</t>
+              
+              <t>Construct a table mapping SSRC to RtpReceiver. For
+              each RtpReceiver configured to receive from this
+              transport, and for each SSRC that RtpReceiver is
+              configured to receive, let the key be the SSRC and the
+              value be the RtpReceiver.  The SSRCs of a given
+              RtpReceiver are found in the a=ssrc lines of the m=
+              section corresponding to that RtpReceiver in the remote
+              description.</t>
+
+              <t>Construct a table mapping Payload Type to
+              RtpReceiver. For each RtpReceiver configured to receive
+              from this transport, and for each Payload Type that
+              RtpReceiver is configured to receive, let the key be the
+              Payload Type and the value be the RtpReceiver.  The
+              Payload Types of a given RtpReceiver are found in the
+              a=fmtp lines of the m= section corresponding to that
+              RtpReceiver in the local description.</t>
```

Comment:
Perhaps the easiest thing to do is say "if there are duplicate RtpReceivers for the same key, use the  RtpReceiver whose m= section appears first in the SDP".   Such a rule would apply also to the table as it is updated over time, not just when it is constructed (which we still need to resolve)